test/verify/check-menu

```diff
@@ -50,5 +52,19 @@ class TestMenu(MachineCase):
             b.enter_page("/system")
             b.wait_visible("#memory_status")
 
+        b.switch_to_top()
+        b.click("a[href='/playground/exception']")
+        b.enter_page("/playground/exception")
+        b.wait_visible("button")
+        for i in range(0, 10):
+            b.click("button")
+            time.sleep(2)
+        else:
+            raise Error("Did not see unhandled exception here")
+
+        b.switch_to_top()
+        b.wait_visible("#navbar-oops")
+        b.wait_visible("#navbar-oops")
```

Comment:
It may be that we remove these lines, if it turns out that in order to catch errors via the CDP driver, we have to disable the Ooops error handling in Cockpit.

Reply:
I just tested this, and this isn't necessary. I posted #9639 that puts back failling on unhandled exceptions, and the check-menu test here now picks this up.